test/integration/migrate/index.js

```diff
@@ -268,12 +221,52 @@ module.exports = function(knex) {
 
   });
 
+  describe('knex.migrate.latest in parallel', function() {
+
+    afterEach(function() {
+      return knex.migrate.rollback({directory: 'test/integration/migrate/test'});
+    });
+
+    if (knex.client.dialect === 'postgres') {
+      it("is able to run two migrations in parallel in postgres", function () {
```

Comment:
We are not running integration tests yet for mssql on CI but eventually we will. So for this pull request would be enough to add also `mssql` to be tested in addition to `postgres`.